konfera/models/ticket.py

```diff
@@ -10,6 +10,11 @@
 CHECKEDIN = 'checked-in'
 CANCELLED = 'cancelled'
 
+REQUESTED = 'requested'
+REGISTERED = 'registered'
+CHECKEDIN = 'checked-in'
+CANCELLED = 'cancelled'
```

Comment:
It seems after the merge you have both your consts and also consts from master. When you started working on it they werent there but in the mean time they were added t o master so now you have them twice. Please remove duplicates.